src/selection.js

```diff
@@ -87,9 +87,21 @@ export default class Selection extends Cursor {
         link.setAttribute(key, value)
       }
     }
+    if (config.linkMarkup.trim) this.trimRange()
+
     this.forceWrap(link)
   }
 
+  // trims whitespaces on the left and right of a selection, i.e. what you want in case of links
+  trimRange () {
+    const textToLink = this.range.toString()
```

Comment:
A bit late but: why `Link` here, shouldn’t the variable be called `textToTrim`?

Reply:
thanks you're absolutely right: https://github.com/livingdocsIO/editable.js/pull/259